### ISSUE TITLE
Alt attributes

### ISSUE THREAD

**Issue by jordanwillis** [aedroos6/prj-rev-bwfs-dasmoto#2]
Don't forget to always include `alt` attributes in all of your `<img>` elements.  This is one of the best ways to ensure your web site adheres to some of the best practice accessibility rules.  Without an `alt` attribute, screen readers and other similar accessibility related tools are unable to convey the meaning or context of the image.

https://github.com/aedroos6/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/index.html#L12
https://github.com/aedroos6/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/index.html#L17
https://github.com/aedroos6/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/index.html#L22
